custom/icds_reports/migrations/0188_new_fields_to_bihar_demogrpahics.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.28 on 2020-05-10 08:23
+from __future__ import unicode_literals
```

Comment:
Migration would need to be udpated as some recent changes is merged to master which will break this migration

Reply:
done